stack-bootstrap.yaml

```diff
@@ -1,4 +1,4 @@
-resolver: lts-9.21
+resolver: lts-10.3
```

Comment:
Does this foreclose any bootstrap options?

Reply:
Not really; as long as we keep testing the older LTS's in TravisCI, we'll be reasonable sure it's possible to bootstrap with older versions by manually editing this file.  I think it's just more convenient for developers if this is in sync with `stack.yaml`.